plugins/sampledata/blog/blog.php

```diff
@@ -1059,7 +1121,7 @@ public function onAjaxSampledataApplyStep3()
 			array(
 				'title'     => Text::_('PLG_SAMPLEDATA_BLOG_SAMPLEDATA_MODULES_MODULE_11_TITLE'),
 				'ordering'  => 4,
-				'position'  => 'cpanel',
+				'position'  => 'cPanel',
```

Comment:
Is this change right? I'm not sure.

Reply:
```suggestion
				'position'  => 'cpanel',
```

No change required
https://github.com/joomla/joomla-cms/blob/eeefb15c78e27a465362abccef64be09b9a449c2/administrator/templates/atum/templateDetails.xml#L43